server/Makefile

```diff
@@ -1,6 +1,7 @@
 SHELL := bash
 TAG ?= server
 RUST_LOG ?= INFO
+include .env
```

Comment:
it is possible to use vars defined in makefile directly to avoid existence of `.env` file?